sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DiagnosticsClientContext.java

```diff
@@ -143,6 +147,18 @@ public DiagnosticsClientConfig withPreferredRegions(List<String> preferredRegion
             return this;
         }
 
+        public DiagnosticsClientConfig withProactiveContainerInitConfig(
+            CosmosContainerProactiveInitConfig config) {
+
+            if (config == null) {
+                this.proactivelyInitializedContainersAsString = "";
+            } else {
+                this.preferredRegionsAsString = config.toString();
```

Comment:
should it be `proactivelyInitializedContainersAsString `?

Reply:
:-) Done